subs/pantry/src/Pantry/Hackage.hs

```diff
@@ -427,7 +428,7 @@ getHackageTarballKey pir@(PackageIdentifierRevision name ver (CFIHash sha _msize
 getHackageTarballKey pir = packageTreeKey <$> getHackageTarball pir Nothing
 
 getHackageTarball
-  :: (HasPantryConfig env, HasLogFunc env)
+  :: (HasPantryConfig env, HasLogFunc env, HasProcessContext env)
```

Comment:
why did this new constraint appear here?

Reply:
That's because `storeTree` function now has that constraint.